src/m365/aad/commands/m365group/m365group-set.ts

```diff
@@ -141,6 +142,11 @@ class AadM365GroupSetCommand extends GraphCommand {
 
   public async commandAction(logger: Logger, args: CommandArgs): Promise<void> {
     try {
+      const isUnifiedGroup = await aadGroup.isUnifiedGroup(args.options.groupId);
```

Comment:
Will this work? We don't have an option like this in this command

```suggestion
      const isUnifiedGroup = await aadGroup.isUnifiedGroup(args.options.id);
```

Reply:
You are correct. My apologies